NavigationAdvancedSample/app/src/main/java/com/example/android/navigationadvancedsample/NavigationExtensions.kt

```diff
@@ -171,7 +171,8 @@ private fun BottomNavigationView.setupDeepLinks(
             containerId
         )
         // Handle Intent
-        if (navHostFragment.navController.handleDeepLink(intent)) {
+        if (navHostFragment.navController.handleDeepLink(intent)
```

Comment:
Just small thing - what about extracting `navHostFragment.navController` to  local variable to avoid duplication - it occurs tree times :-)

Reply:
Happy to accept this as a follow on pull request.